ansible/roles/plpsql/templates/postgresql.conf

```diff
@@ -76,14 +76,14 @@ max_connections = 100			# (change requires restart)
 # - Security and Authentication -
 
 #authentication_timeout = 1min		# 1s-600s
-#ssl = off				# (change requires restart)
-#ssl_ciphers = 'HIGH:MEDIUM:+3DES:!aNULL' # allowed SSL ciphers
-					# (change requires restart)
-#ssl_prefer_server_ciphers = on		# (change requires restart)
-#ssl_ecdh_curve = 'prime256v1'		# (change requires restart)
-#ssl_cert_file = 'server.crt'		# (change requires restart)
-#ssl_key_file = 'server.key'		# (change requires restart)
-#ssl_ca_file = ''			# (change requires restart)
+
+ssl = on
+ssl_ciphers = 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'
+ssl_prefer_server_ciphers = on
+ssl_cert_file = '/srv/pl-psql/ssl/{{ inventory_hostname }}.cert'
+ssl_key_file = '/srv/pl-psql/ssl/{{ inventory_hostname }}.key'
+ssl_ca_file = '/srv/pl-psql/ssl/pg_user_ca.cert'
```

Comment:
I'd say, it looks like a bad idea to put secret key in PGDATA as soon as we're going to upload snapshot of PGDATA directory to S3 monthly.

Reply:
Aren't we going to be uploading only certain files from that directory?

In any case this approach of putting a subdirectory in there turned out to not work for other reasons as well so it's changed in the latest commit.